### ISSUE TITLE
When building starting Queue, commander can build himself into a building.

### ISSUE THREAD

**Issue by Akslaton** [beyond-all-reason/Beyond-All-Reason#1257]
![Commander Building into Building](https://user-images.githubusercontent.com/91833558/190555686-10a9edfa-3ab9-4953-8ff6-ef6f60da6a91.jpg)

After choosing your starting location you can begin queuing starting buildings before you ready up.

So, at the beginning of the game, if you queue a building where your commander will drop in, he will build the building without first moving out of the way, thus trapping himself inside the structure until it is destroyed or reclaimed.
At any other time of the game, the building unit moves out of the way before starting a structure, and it will not build where another unit is sitting. I'm not really sure how or why this is different.

I hope this bug report is helpful
thank you for collaborating and creating such a great game.

**Comment by 6AKU66** [comment]
Hi, it's duplicate of #803

**Comment by Akslaton** [comment]
I'm guessing I should just close this issue since it's a duplicate of #803 

I apologize for my ignorance, this is like my second actual issue report on Github. 
I'm very new to actually participating in the open source community.

**Comment by 6AKU66** [comment]
> I'm guessing I should just close this issue since it's a duplicate of #803
> 
> I apologize for my ignorance, this is like my second actual issue report on Github. I'm very new to actually participating in the open source community.

It's ok, duplication reports always gonna happen from time to time.
